src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx

```diff
@@ -25,7 +25,17 @@ export const ActivityGroupAmountChange: FC<Props> = ({ nonZeroAmounts, textSize
 
   const children = useMemo(() => {
     if (nonZeroAmounts.length === 0) {
-      return null;
+      return (
```

Comment:
It looks like '---' will be displayed in case there are absolutely no tokens deltas, even with absent token metadata. Is it just as planned?

Reply:
Yep
https://www.figma.com/file/n4rBy6iCiwxavYIV08Am71/Mobile-UI-Temple-Wallet?type=design&node-id=20347-9815&mode=design&t=uFygpr5Pcp2l7OsM-0